code/game/objects/items.dm

```diff
@@ -83,14 +83,7 @@
 	var/reach = 1
 
 
-	/* Species-specific sprites, concept stolen from Paradise//vg/.
-	ex:
-	sprite_sheets = list(
-		"Tajara" = 'icons/cat/are/bad'
-		)
-	If index term exists and icon_override is not set, this sprite sheet will be used.
-	*/
-
+	/// Species-specific sprites, concept stolen from Paradise//vg/.
```

Comment:
Why

Reply:
Autodoc, I updated it to include an example and the index term info.